top-level-walruses/piet/cli.py

```diff
@@ -0,0 +1,80 @@
+from io import BytesIO
+from pathlib import Path
+from typing import Annotated, Optional
+
+import typer
+from PIL import Image
+
+from .generator import ImageGenerator
+from .interpreter import PietInterpreter
+
+app = typer.Typer()
+
+
+@app.command()
+def generate(
+    input_path: Path,
+    output_path: Path,
+    cols: Optional[int] = None,
```

Comment:
This is minor, but the *slightly* recommended method of this type hint would be `int | None`.